trusted/script/generateSession.js

```diff
@@ -183,19 +186,21 @@ function generateTable(tableBody, sessionID, password, status, timestamp, counte
         data: JSON.stringify({session: sessionID, password: password, last_fetch: timestamp}),
         dataType: "json",
         success: function (data) {
```

Comment:
Do not show the user key (participation code) in the submission tracking interface.